cdxev/__main__.py

```diff
@@ -951,5 +987,14 @@ def invoke_build_public_bom(args: argparse.Namespace) -> int:
     return Status.OK
 
 
+def invoke_init_sbom(args: argparse.Namespace) -> int:
+    name = args.name
+    version = args.version
+    supplier = args.supplier
+    sbom = initialize_sbom(software_name=name, supplier_name=supplier, version=version)
```

Comment:
why not directly assign `name`, `version` and `supplier` in this line? Seems odd to assign the values to variables, which are solely used for the function.

Reply:
done